packages/patterns/src/types.js

```diff
@@ -29,9 +29,38 @@ export {};
  * CopyTaggeds recognized as Keys.
  *
  * Distributed equality is location independent.
- * The same two Keys, passed to another location, will be `keyEQ` there iff
- * they are `keyEQ` here. (`keyEQ` tests equality according to the
+ * The same two Keys, passed to another location, will be {@link keyEQ} there iff
+ * they are {@link keyEQ} here. ({@link keyEQ} tests equality according to the
  * key distributed equality semantics.)
+ *
```

Comment:
I'm wondering if the rank/key order details would fit better in CONTRIBUTING.md along with the other details extracted from README. I don't think I'd look for details like that in a types.js file (although tbh I'm not sure "CONTRIBUTING.md" is where I'd look either, maybe something like IMPLEMENTATION-DETAILS.md).

Reply:
I had it in CONTRIBUTING for a bit, but rankCompare and keyCompare are part of the api, not an implementation detail.

The material could perhaps move to those functions, but I suspect the editorial work would exceed my timebox for this.